cmd/metrics-powerflex/main.go

```diff
@@ -144,10 +215,16 @@ func main() {
 		storagePoolTickInterval = time.Duration(numSeconds) * time.Second
 	}
 
+	config.SDCTickInterval = sdcTickInterval
+	config.VolumeTickInterval = volumeTickInterval
+	config.StoragePoolTickInterval = storagePoolTickInterval
```

Comment:
After updating the tick intervals, we also need a way to call Reset on each ticker so that the new value is applied (https://golang.org/pkg/time/#Ticker.Reset)

Reply:
Ticker.Reset only works from Go version 1.15, we are still supporting 1.14. The only option seems to be creating a new ticker when the interval has changed.

Reply:
fixed